Cargo.toml

```diff
@@ -223,7 +223,6 @@ fs2 = { git = "https://github.com/tikv/fs2-rs", branch = "tikv" }
 # Remove this when a new version is release. We need to solve rust-lang/cmake-rs#143.
 cmake = { git = "https://github.com/rust-lang/cmake-rs" }
 
-sysinfo = { git = "https://github.com/tikv/sysinfo", branch = "0.26-fix-cpu" }
```

Comment:
The latest version of `sysinfo` has already merged this pr (https://github.com/tikv/sysinfo/commit/5a1bcf08816979624ef2ad79cfb896de432a9501) and fixed this issue, and this patch is redundant.